db/migrate/20220513214928_convert_database_to_utf8_mb4.rb

```diff
@@ -0,0 +1,63 @@
+class ConvertDatabaseToUtf8Mb4 < ActiveRecord::Migration
+  def up
+    connection.execute "ALTER DATABASE #{ActiveRecord::Base.connection.current_database} CHARACTER SET 'utf8mb4' COLLATE 'utf8mb4_unicode_520_ci';"
+    
+    %w{
```

Comment:
My intention was to be explicit about the tables _we know_, since we shouldn't go around modifying tables out of our control. But I'm not really sure on the way to go.

I feel this `ar_internal_metadata` table issue is a nice evidence to inform this decision, but I'm not sure _in which way_.

@ysbaddaden ?

Reply:
Yes, they are. That's where I've sourced the list from.